src/app/home-page/widget/organization-box/organization-box.component.ts

```diff
@@ -19,10 +20,11 @@ import { AlertService } from 'app/shared/alert/state/alert.service';
 export class OrganizationBoxComponent extends Base implements OnInit {
   Dockstore = Dockstore;
   listOfOrganizations: Array<OrganizationUpdateTime> = [];
-  firstCall = true;
+  pendingRequests$: Observable<Array<OrganizationUser>>;
+  pendingInvites$: Observable<Array<OrganizationUser>>;
+  filterText: string;
   totalOrgs: number = 0;
-  pendingRequests: Array<OrganizationUser> = [];
-  pendingInvites: Array<OrganizationUser> = [];
+  firstCall: Boolean = true;
```

Comment:
https://stackoverflow.com/questions/64443288/what-is-the-difference-between-boolean-and-boolean-in-typescript -- you should use `boolean` instead.

Reply:
Ah neat, didn't know that thanks